kythe/extractors/bazel/extract.sh

```diff
@@ -50,5 +50,6 @@
 # Collect any extracted compilations.
 mkdir -p "$KYTHE_OUTPUT_DIRECTORY"
 find bazel-out/*/extra_actions/external/kythe_release -name '*.kzip' | \
-  xargs /kythe/tools/kzip merge --output "$KYTHE_OUTPUT_DIRECTORY/compilations.kzip"
+  xargs -r /kythe/tools/kzip merge --output "$KYTHE_OUTPUT_DIRECTORY/compilations.kzip"
+test -f "$KYTHE_OUTPUT_DIRECTORY/compilations.kzip" || exit $?
```

Comment:
Put this last and remove the `|| exit $?`

Reply:
Done.